docs/plugin-development/restrictions.md

```diff
@@ -82,6 +82,44 @@ start debugging performance issues is through the Plugin Statistics window,
 which can be found through Plugins > Open Plugin Stats in the dev menu
 (`/xldev`).
 
+## Can my plugin talk to a server I run?
+
+Plugins are permitted to communicate with special backend servers, though there
+are certain considerations and requirements that must be met:
+
+- Plugins should take care to send the minimum amount of data necessary to do
+  their job. Whenever feasible, plugins should hash information like Content IDs
+  or player names on the client side so that a server-side data breach does not
+  reveal information.
+- Plugins collecting information for telemetry or analytics purposes must capture
+  as little information as necessary, and must clearly explain to the user what is
+  being collected and why.
+  - Users should be required to opt in to telemetry information, but this may be
+    done as part of a "welcome to this plugin" experience or controlled by a
+    Dalamud setting.
+- Plugins should use secure communication (e.g. HTTPS, TLS, SSL) where possible,
+  and should have certificates issued from a trusted certificate authority such
+  as [Let's Encrypt](https://letsencrypt.org/). Plugins should connect to servers
```

Comment:
I honestly think this should be a hard requirement - Domain names are relatively affordable and using LetsEncrypt for SSL is easy and free.

Reply:
Ah, I forgot to update the comment. I've updated the proposal to require SSL now.

I'm still a bit *mmmm* about requiring a domain name, since it is a cost burden on developers who may not necessarily be able to afford it, but I suppose there are services like afraid.org and NoIP.

Something to note: plugin devs do ***not*** need to require SSL if they're implementing custom home servers. This requirement only applies to the officially-run and officially-shipped default server.